### DISCUSSION TITLE
"View what others see" shows different Repository count between tabs

### DISCUSSION THREAD

**Discussion by tommy-gilligan** [community/community#19736] [Other Feature Feedback, Questions, & Ideas]
On profile tab it is a count of just public repositories
<img width="1104" alt="Screen Shot 2022-06-30 at 11 27 08" src="https://user-images.githubusercontent.com/7865781/176573191-6b7a5b0e-a0b2-431d-99a4-e748883a0fc6.png">

On repository tab it is a count of public + private repositories
<img width="1128" alt="Screen Shot 2022-06-30 at 11 27 11" src="https://user-images.githubusercontent.com/7865781/176573193-a62bfcc4-bcd5-48e9-be56-bc326715c140.png">

I think for both of these tabs, it should be a count of public repositories only.

- I have Make profile private and hide activity Beta enabled
- and Include private contributions on my profile disabled

**Comment by thuongtruong1009** [comment]
Hi @tomgilligan 
Well, now you don't have to worry. Because the count and information of the private repo are only visible to you, so you can manage and maintain it. As for other people viewing your profile, they can't see your private repositories information and count, instead of they only see the public repositories.

For example, I went to look at your profile and I won't see private repositories, I only see public repositories.
![image](https://user-images.githubusercontent.com/71834167/176576926-db6e9399-edd9-4905-a6ec-1ccc602cdc99.png)

Hope it helps you!
Have a nice day🥰

**Comment by tommy-gilligan** [comment]
Thank you @thuongtruong1009 
This is feedback for a beta feature.  I think they should still fix the wrong count, it is confusing.

**Reply by thuongtruong1009** [reply]
Oh, I'm glad I discussed it with you. And can you help me ***mark*** ***it*** ***as*** ***an*** ***answer*** for all to know and for other Github developers to fix this problem in the future
Thank you very much! 🥰
